<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>113</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>113</y>
     <width>1920</width>
     <height>967</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>14</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAgregar">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>175</y>
     <width>278</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;



</string>
   </property>
   <property name="text">
    <string> Agregar Equipos</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>c.png</normaloff>c.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>36</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>MANTENIMIENTO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>36</y>
     <width>126</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1836</x>
     <y>29</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>perfil-de-usuario.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>190</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Equipos: {total} (? conectados)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>255</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>970</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonEditar1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>990</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>edit.png</normaloff>edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="botonSacar1">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>990</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>close.png</normaloff>close.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>180</y>
     <width>598</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: white;

border: 2px solid black;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1261</x>
     <y>198</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;

border: 2px solid white;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;

font-size:15px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>180</y>
     <width>421</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(0, 45, 107);

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Ordenar por:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="botonOrdenar1">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>186</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8;
font-size:20px;
color:black;


border: 2px solid #e8e8e8;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Seleccionar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>down_arrow.png</normaloff>down_arrow.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBorrar">
   <property name="geometry">
    <rect>
     <x>1771</x>
     <y>198</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: white;

border: 2px solid white;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>close2.png</normaloff>close2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBuscar">
   <property name="geometry">
    <rect>
     <x>1221</x>
     <y>193</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: white;

border: 2px solid white;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>search.png</normaloff>search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="botonLogOut">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>30</y>
     <width>199</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: black;
background-color: #ebebeb;</string>
   </property>
   <property name="text">
    <string>Cerrar Sesión</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAdmin">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>30</y>
     <width>199</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #000000;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;

color: black;
background-color: #ebebeb;</string>
   </property>
   <property name="text">
    <string>Administración</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>256</y>
     <width>1800</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 2px solid #ffffff;
    border-radius: 30px; /* Bordes redondeados */
    padding: 5px;</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>botonAgregar</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_7</zorder>
  <zorder>label_14</zorder>
  <zorder>label</zorder>
  <zorder>label_20</zorder>
  <zorder>botonEditar1</zorder>
  <zorder>botonSacar1</zorder>
  <zorder>label_24</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_22</zorder>
  <zorder>botonOrdenar1</zorder>
  <zorder>botonBorrar</zorder>
  <zorder>botonBuscar</zorder>
  <zorder>botonLogOut</zorder>
  <zorder>botonAdmin</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
